utils/network.go

```diff
@@ -1,3 +1,16 @@
+// Copyright 2015 Tigera Inc
```

Comment:
Strictly speaking I think this should say 2018

Reply:
should it be?  it's old code.

Reply:
I believe since the file is being updated in 2018 (even just to add the copyright) it should say 2018.

Reply:
ok, i bumped the year. thank you.